### PR TITLE
add postgresql questions

### PR DESCRIPTION
**Pull request by mandeep-s1ngh**
## Summary of changes

I have added two postgresql questions and successfully tested them with the documentation links locally.
One question was about the syntax for deleting a database and the other was about the default port number.

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [ ] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

File: src/data/sql-quiz.ts [hunk 1]
```diff
@@ -1099,6 +1099,25 @@ const sqlQuiz = [
...
     Link: "https://www.freecodecamp.org/news/the-sql-having-clause-explained-with-example-syntax/"
+  },
+  {
+    Question: "In PostgreSQL, what is the syntax for deleting a database?",
+    Answer: "DROP DATABASE database_name;",
+    Distractor1: "DELETE DATABASE database_name;",
+    Distractor2: "REMOVE DATABASE database_name;",
+    Distractor3: "DESTROY DATABASE database_name;",
+    Explanation:
+      "In PostgreSQL, The DROP DATABASE command is used to permanently delete an entire database.",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:1110]:
```suggestion
      "In PostgreSQL, the DROP DATABASE command is used to permanently delete an entire database.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by mandeep-s1ngh** [comment]
Hi, thank you for your quick response. I have committed the change as requested.
Almost had a perfect contribution 😅

File: src/data/sql-quiz.ts (hunk 1 above)
Comment (mandeep-s1ngh) [reply src/data/sql-quiz.ts]: Hi, thank you for your quick response. I have committed the change as requested.
Almost had a perfect contribution 😅

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**